src/app/Fake.Core.Process/ProcessUtils.fs

```diff
@@ -7,14 +7,27 @@ open Fake.IO.FileSystemOperators
 
 [<RequireQualifiedAccess>]
 module ProcessUtils =
+    /// Information about a dotnet tool
+    type DotNetTool =
+      {
+        DotNetCli : string option
+        Tool : string option
+      }
+    with static member Create() = {DotNetCli = None; Tool = None}
+
+    /// Select which style of tool implementation
+    type ToolType =
+      | DotNet of DotNetTool
+      | Global
```

Comment:
Does this mean "Global Tool" or something else? If yes, should we rename "DotNetTool" to "DotNetLocalTool"?

Reply:
Saying `DotNet` here covers anything initiated by `dotnet toolname`, be it a .net core 3.0 local tool or a pre-3.0 `DotNetCliToolReference` tool.